enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/facelets/CompositeComponentLibrary.java

```diff
@@ -250,9 +250,11 @@ private synchronized void load() {
                 for (Map<String, String> attrsMap : model.getExistingInterfaceAttributes()) {
                     String attrname = attrsMap.get("name"); //NOI18N
                     boolean required = Boolean.parseBoolean(attrsMap.get("required")); //NOI18N
-                    String attributeDescription = getAttributesDescription(model, true);
+                    String attributeDescription = attrsMap.get("shortDescription"); //NOI18N
```

Comment:
Could you please describe what this change means? A visualization would be helpful.

Reply:
With `getAttributesDescription` you used to get the description of all attributes on each individual attribute, without any distinction. Hence the full list all over the place. By using `shortDescription`, you really get just the description of the corresponding attribute, which is what you actually want to see.